src/pages/ProposalsSpam.jsx

```diff
@@ -0,0 +1,221 @@
+import React, { useEffect, useState, Fragment } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import {
+  Box,
+  Table,
+  Thead,
+  Tr,
+  Th,
+  Tbody,
+  Td,
+  IconButton,
+  Flex,
+} from '@chakra-ui/react';
+import { RiRefreshLine } from 'react-icons/ri';
+
+import { useTX } from '../contexts/TXContext';
+import useBoost from '../hooks/useBoost';
+import MainViewLayout from '../components/mainViewLayout';
+import ContentBox from '../components/ContentBox';
+import ListSelect from '../components/listSelect';
+import { timeToNow } from '../utils/general';
+import { fetchAllActivity } from '../utils/theGraph';
+import { SPAM_FILTER_UNSPONSORED } from '../graphQL/dao-queries';
+import { proposalResolver } from '../utils/resolvers';
+
+const FILTERS = {
+  main: [
+    { name: 'All', value: 'all' },
+    { name: 'Unsponsored', value: 'Unsponsored' },
+    { name: 'Cancelled', value: 'Cancelled' },
+  ],
+};
+
+const proposalRow = (proposal, daochain, daoid) => {
+  return (
+    <Fragment key={proposal.proposalId}>
+      <Tr>
+        <Td border='none'>{proposal.proposalId}</Td>
+        <Td border='none'>{proposal.sharesRequested}</Td>
+        <Td border='none'>{proposal.lootRequested}</Td>
+        <Td border='none'>
+          {`${proposal.paymentRequested /
```

Comment:
Do we have utils for cleaning these decimals things up?

Reply:
:heavy_check_mark: